scripts/plugin_runner/utils.py

```diff
@@ -0,0 +1,30 @@
+from argparse import ArgumentParser
+
+from utils.models.cli_models import TaskFlagArgs
+
+
+def configure_parser(parser: ArgumentParser):
```

Comment:
Could you please move this file to utils folder?

Reply:
I am not sure about it because it allows configuring only parsers, not any other scripts